src/query-builder/InsertQueryBuilder.ts

```diff
@@ -356,6 +356,7 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
         overwrite: string[],
         conflictTarget?: string | string[],
         orUpdateOptions?: InsertOrUpdateOptions,
+        conflictConditions?: string[],
```

Comment:
too many parameters already... at some point we need to add an overload to accept both parameters and object with parameters.

Reply:
I agree as well. The reason I added a new optional function parameter was make the smallest change that wouldn't break the preferred calling method (as listed in the deprecation message)
```
     * `.orUpdate({ columns: [ "is_updated" ] }).setParameter("is_updated", value)`
     *
     * is now `.orUpdate(["is_updated"])`
     *
     * `.orUpdate({ conflict_target: ['date'], overwrite: ['title'] })`
     *
     * is now `.orUpdate(['title'], ['date'])`
```
If there is a preferred overloading setup elsewhere in the repo I can adapt it to suit this.